### PR TITLE
Wrong order of comments in default messages config file

### PR DESCRIPTION
**Pull request by alexandernst**
The generated file with the command `./yii message/config path/to/config.php` contains comments in the wrong order.

### CODE REVIEW

File: framework/views/messageConfig.php [hunk 1]
```diff
@@ -69,11 +69,7 @@
     /*
     // 'po' output format is for saving messages to gettext po files.
     'format' => 'po',
-    // Root directory containing message translations.
-    'messagePath' => __DIR__ . DIRECTORY_SEPARATOR . 'messages',
     // Name of the file that will be used for translations.
     'catalog' => 'messages',
-    // boolean, whether the message file should be overwritten with the merged messages
```
Comment (samdark) [inline framework/views/messageConfig.php]: Why it was removed?

File: framework/views/messageConfig.php [hunk 2]
```diff
@@ -69,11 +69,7 @@
     /*
     // 'po' output format is for saving messages to gettext po files.
     'format' => 'po',
-    // Root directory containing message translations.
```
Comment (samdark) [inline framework/views/messageConfig.php]: Why it was removed?

Comment (alexandernst) [inline framework/views/messageConfig.php]: Because this same config entry appears around 15 lines above the removed line.

**Comment by samdark** [comment]
Applied partly. Thanks!

**Pull request closed without merging**